src/main/java/com/alphawallet/attestation/core/DERUtility.java

```diff
@@ -104,30 +104,19 @@ public static BigInteger decodeSecret(byte[] secretBytes) {
     return outstream.toByteArray();
   }
 
-  public static String printDER(byte[] input, String type) {
+  public static byte[] toPEM(byte[] input, String type) {
     try {
       Base64Encoder coder = new Base64Encoder();
       ByteArrayOutputStream outstream = new ByteArrayOutputStream();
       coder.encode(input, 0, input.length, outstream);
       byte[] encodedCert = outstream.toByteArray();
-      StringBuilder builder = new StringBuilder();
-      builder.append("-----BEGIN " + type + "-----\n");
-      addBytes(builder, encodedCert);
-      builder.append("-----END " + type + "-----");
-      return builder.toString();
+      ByteArrayOutputStream outputStream = new ByteArrayOutputStream( );
+      outputStream.write(("-----BEGIN " + type + "-----\n").getBytes(StandardCharsets.UTF_8));
+      outputStream.write(encodedCert);
```

Comment:
No new-line information gets encoded which is needed for valid PEM files the results in the command line demo not working anymore.

Reply:
new line added thanks.